### PR TITLE
chore(package): update dev dependencies

### PR DESCRIPTION
**Pull request by philbooth**
~~When I run `grunt release` in this repo I see a bunch of errors~~

~~So~~ I thought I'd try and update the dev deps ~~to see if it makes them go away. It didn't but maybe we want the updates anyway~~, hence this PR. Feel free to close if it's not wanted.

There's a secondary commit containing lint fixes, which are mostly indentation so you may want to review this with [`?w=1`](https://github.com/mozilla/fxa-js-client/pull/299/files?w=1).

**EDIT**: Errors were just because I didn't `npm run setup`, you can ignore that part now.

@mozilla/fxa-devs r?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,20 +35,23 @@
...
+    "eslint-plugin-scanjs-rules": "0.2.1",
+    "grunt": "1.0.3",
+    "grunt-build-control": "0.7.1",
+    "grunt-bump": "0.8.0",
+    "grunt-bytesize": "0.2.0",
+    "grunt-cli": "1.3.2",
+    "grunt-contrib-clean": "2.0.0",
+    "grunt-contrib-watch": "1.1.0",
+    "grunt-contrib-yuidoc": "1.0.0",
+    "grunt-conventional-changelog": "6.1.0",
```
Comment (vladikoff) [inline package.json]: this might break the changelog structure, or could be fine :)

Comment (philbooth) [reply package.json]: Ah yeah, it does. Didn't test that one, sorry. I'll put it back.

Comment (philbooth) [reply package.json]: ...although I get the same error with `3.0.0` too:

```
No "conventionalChangelog" targets found.
```

Is it expected or is something wrong?

**Comment by philbooth** [comment]
I've pushed two more commits to this branch, probably warrants a re-review.

* In cd97a20, I stopped the release script from autopushing. It still autopushes the docs.

* In 54fc2e2, I add an `npm run release` script that does `npm run setup` before `grunt release`.

@mozilla/fxa-devs r?

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "test": "grunt test",
     "test-local": "intern-client config=tests/intern auth_server=LOCAL",
     "setup": "npm install && grunt sjcl",
-    "contributors": "git shortlog -s | cut -c8- | sort -f > CONTRIBUTORS.md"
+    "contributors": "git shortlog -s | cut -c8- | sort -f > CONTRIBUTORS.md",
+    "release": "npm run setup && grunt release"
```
Comment (shane-tomlinson) [inline package.json]: 👍

**Comment by shane-tomlinson** [comment]
@philbooth can you migrate this PR to the monorepo?

**Pull request closed without merging**